src/test/java/pl.noip.lolstats.lol.stats/service/ChampDataTest.java

```diff
@@ -0,0 +1,26 @@
+package pl.noip.lolstats.lol.stats.service;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+@RunWith(SpringRunner.class)
+@ActiveProfiles("test")
+public class ChampDataTest {
+
+    @Autowired
+    private ChampionServiceImpl championServiceImpl;
+
+    @Test
+    public void getjsondata() {
```

Comment:
test name should say, what is tested. Eg. ShouldReturnChampionNameForChampionKey